include/grpcpp/impl/codegen/server_interface.h

```diff
@@ -174,13 +174,14 @@ class ServerInterface : public internal::CallHook {
     bool done_intercepting_;
   };
 
+  /// RegisteredAsyncRequest is not part of the C++ API
```

Comment:
What does this mean?  Isn't this implicit, given that it's in the `internal` namespace?

Reply:
It's not internal. This is a protected member of `grpc::ServerInterface` which is API. These request classes should be cleaned up a bit. I don't think there are any external users but this should become part of an API cleanup.